egs/wsj/s5/utils/subword/prepare_lang_subword.py

```diff
@@ -176,7 +168,7 @@
     > $tmpdir/phone_map.txt
 else
   if "$silprob"; then
-    echo "Currently do not support word-dependent silence probability"
+    echo "$0: Currently we do not support word-dependent silence probability" && exit 1;
```

Comment:
Is this check necessary? I see that you already checked this in line 109 and then again in line 146. I guess this code (and the if block at line 146) would never run?

Reply:
I see similar blocks later too

Reply:
Yes, this check is not necessary. I left it here only to remind me that I may add some function here to support word-dependent silence probability in the future.